src/plugins/intel_gpu/src/runtime/execution_config.cpp

```diff
@@ -133,13 +135,21 @@ void ExecutionConfig::apply_rt_info(const IRemoteContext* context, const ov::RTM
 }
 
 void ExecutionConfig::apply_model_specific_options(const IRemoteContext* context, const ov::Model& model) {
-    const auto is_LLM = ov::op::util::is_large_language_model(model) ||
-                        ov::op::util::has_op_with_type<ov::intel_gpu::op::KVCache>(model.shared_from_this());
-    apply_rt_info(context, get_rt_info(model), is_LLM);
+    auto is_paged_attention_model = false;
+    const auto is_LLM = ov::op::util::is_large_language_model(model, [&is_paged_attention_model](std::shared_ptr<ov::Node> node) {
```

Comment:
why do we need this callback?
maybe it's better to return a structure { bool is_LLM, bool is_Paged_Attention; } from "is_large_language_model" helper? or use tuple?